src/app/helptext/network/interfaces/interfaces-list.ts

```diff
@@ -31,4 +33,12 @@ ha_enabled_edit_msg: T("Editing interfaces while HA is enabled is not allowed.")
 
 ha_enabled_delete_title: T("Cannot Delete while HA is Enabled"),
 ha_enabled_delete_msg: T("Deleting interfaces while HA is enabled is not allowed."),
+
+checkin_text: T("Once applied, changes will revert after "),
```

Comment:
"Once" can be misinterpreted as "one time", prefer "When" or "After".  But also try to avoid the if/then sentence structure with the pause in the middle.
```suggestion
checkin_text: T("Changes that have been applied will revert after "),
```